engine/crates/composition/src/compose/context.rs

```diff
@@ -64,8 +73,36 @@ impl<'a> Context<'a> {
         enum_id: federated::EnumId,
         value: StringWalker<'_>,
         deprecation: Option<Option<StringWalker<'_>>>,
+        description: Option<StringWalker<'_>>,
     ) {
-        self.ir.insert_enum_value(enum_id, value, deprecation);
+        let mut composed_directives = Vec::new();
```

Comment:
The function moved to remove one level of unnecessary indirection. Only the description lines are new.